src/Laravel/ApiPlatformProvider.php

```diff
@@ -1193,10 +1195,12 @@ private function registerGraphQl(Application $app): void
 
         $app->singleton(ResolverProvider::class, function (Application $app) {
             $resolvers = iterator_to_array($app->tagged('api_platform.graphql.resolver'));
+            $taggedItemResolvers = iterator_to_array($app->tagged(QueryItemResolverInterface::class));
+            $taggedCollectionResolvers = iterator_to_array($app->tagged(QueryCollectionResolverInterface::class));
 
             return new ResolverProvider(
                 $app->make(GraphQlReadProvider::class),
-                new ServiceLocator($resolvers),
+                new ServiceLocator([...$resolvers, ...$taggedItemResolvers, ...$taggedCollectionResolvers]),
```

Comment:
Note that we recommend to use state providers instead of resolvers for most usages.